hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/conf/RatisClientConfig.java

```diff
@@ -26,9 +29,9 @@
  * RaftClient creation.
  *
  */
-@ConfigGroup(prefix = "raft.client")
+@ConfigGroup(prefix = RatisHelper.HDDS_DATANODE_RATIS_CLIENT_PREFIX_KEY)
 public class RatisClientConfig {
-  @Config(key = "async.outstanding-requests.max",
+  @Config(key = RaftClientConfigKeys.Rpc.REQUEST_TIMEOUT_KEY,
```

Comment:
RaftClientConfigKeys.Rpc.REQUEST_TIMEOUT_KEY,  the config key is wrong. Can you please fix it as it denotes the max async requests count on client?

Reply:
This is taken care of in HDDS-2988.